os/kernel/debug/secure_log.c

```diff
@@ -0,0 +1,127 @@
+#include <stdint.h>
+#include <stdbool.h>
+#include <debug.h>
+
+/***************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Function Prototypes
+ ****************************************************************************/
+
+/****************************************************************************
+ * Global Variables
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Types
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private data
+ ****************************************************************************/
+static int seclog_enable = 0;
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: write_seclog
+ *
+ * Description:
+ *   This function write the value of secure log variable (which is saved in 
+ *   secure storage) in seclog_enable.
+ *
+ * Inputs:
+ *   None
+ *
+ * Return Value:
+ *   None
+ *
+ * Assumptions:
+ *   None
+ *
+ ****************************************************************************/
+void write_seclog(void)
+{
+        sl_ctx hnd;
+        uint32_t ss_idx = 32;
+        hal_data output;
+
+        /* init */
+        if (SECLINK_OK != sl_init(&hnd)) {
+                printf("[write_seclog] Failed to sl_init\n");
+                return;
+        }
+
+        output.priv = NULL;
+        output.priv_len = 0;
+        output.data_len = 2;
+        output.data = (char *)malloc(output.data_len);
+
+        if (SECLINK_OK != sl_read_storage(hnd, ss_idx, &output)) {
+                printf("[write_seclog] Failed to sl_read_storage\n");
+                return;
+        }
+
+        if (!strncmp(output.data, "1", 2)) {
+                seclog_enable = 1;
+        } else {
+                seclog_enable = 0;
+        }
+
+        printf("write_seclog : %d\n", seclog_enable);
+}
+
+/****************************************************************************
+ * Name: read_seclog
+ *
+ * Description:
+ *   This function returns the seclog_enable.
+ *
+ * Inputs:
+ *   None
+ *
+ * Return Value:
+ *   0(disable) or 1(enable)
+ *
+ * Assumptions:
+ *   None
+ *
+ ****************************************************************************/
+int read_seclog(void)
```

Comment:
We will use this to determin log on / off. But I don't think we should name this as `seclog`.
How about `seclevel` or something else? If we use another name, then we can use this at other module when it needs.